src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -24,20 +24,34 @@ const SendRequest = () => {
   const currentStage = STAGES.find(({ key }) => key === request.data?.stage);
 
   let draftErrors;
-
   if (currentStage?.key === "Draft" && request.data) {
     draftErrors = validateRequest(request.data);
   }
 
   const updateHandler = () => {
     if (request.data && currentStage?.next) {
```

Comment:
Is checking for a next stage okay here?  It would prevent any substages from being in the final stage, which should be okay, because the final stage is "Completed" so there should be no substages in it.  If so, we should probably remove subStages from the ENDSTAGE type as none should be defined.